app/models/edition/timestamps.rb

```diff
@@ -0,0 +1,58 @@
+class Edition::Timestamps
+  def self.edited(
+    edition,
+    payload,
+    previous_live_version = nil,
+    now = Time.zone.now
+  )
+    edition.temporary_first_published_at = previous_live_version&.first_published_at
+    edition.temporary_last_edited_at = now
+    edition.publisher_first_published_at = payload[:first_published_at]
+
+    unless edition.update_type == "major"
```

Comment:
Might be worth adding a quick comment here as to why we set this when not major as the code reads a little confusing (as the situation is a bit confusing)